src/deploymentmanager.ts

```diff
@@ -420,11 +420,11 @@ export class DeploymentManager implements IDeploymentManager {
         } else if (this._template.parameters.deploymentId) {
             this._parameters.deploymentId = { value: answers.deploymentId };
         }
-        if (answers.diagnosticsEndpointUrl) {
-            if (this._parameters.diagnosticsEndpointUrl) {
-                this._parameters.diagnosticsEndpointUrl.value = answers.diagnosticsEndpointUrl;
-            } else if (this._template.parameters.diagnosticsEndpointUrl) {
-                this._parameters.diagnosticsEndpointUrl =  { value: answers.diagnosticsEndpointUrl };
+        if (answers.appInsightsInstrumentationKey) {
+            if (this._parameters.appInsightsInstrumentationKey) {
```

Comment:
Can you add this value to parameters.json?

Reply:
added standard-parameters.json